src/context.rs

```diff
@@ -523,16 +531,20 @@ impl AudioContext {
         }
     }
 
-    /// Closes the `AudioContext`, releasing the system resources being used. This will not
-    /// automatically release all `AudioContext`-created objects, but will suspend the progression of
-    /// the currentTime, and stop processing audio data.
+    /// Closes the `AudioContext`, releasing the system resources being used.
+    ///
+    /// This will not automatically release all `AudioContext`-created objects, but will suspend
+    /// the progression of the currentTime, and stop processing audio data.
```

Comment:
So the nodes renderers will never be dropped, if `graph::render` is never called anymore? (that's really not a big deal for now I guess)

Reply:
To be 100% sure we could write a `Drop` implementation for the `GainProcessor` for example that will `println` when it is dropped. Then we can check the simple example and see if what I am saying is actually true ;)